content/kubermatic/master/upgrading/2.14_to_2.15/chart_migration/_index.en.md

```diff
@@ -0,0 +1,282 @@
+1. Start migrating Seeds:
+   1. Pick a Seed
+   1. Uninstall `kubermatic` Helm chart from the seed cluster.
+   1. Update the `Seed` resource in the master and remove the `operator.kubermatic.io/skip-reconciling` annotation.
+   1. Wait for the operator to reconcile the seed cluster.
+   1. Continue with the next seed cluster.
+
+{{% notice info %}}
+This poses a good opportunity to also make the switch to Helm 3. All other charts for the KKP
+stack (cert-manager, nginx-ingress-controller, etc.) can be migrated in-place from Helm 2 to 3.
+{{% /notice %}}
+
+The following sections describe the steps above in greater detail.
+
+#### Backups
+
+Backing up Kubermatic resources can be done in many ways, for example simply by using `kubectl`
+and writing the YAML into local files:
+
+```bash
+kubectl get addons -A -o yaml > addons.yaml
+kubectl get addonconfigs -A -o yaml > addonconfigs.yaml
+kubectl get admissionplugins -A -o yaml > admissionplugins.yaml
+kubectl get clusters -A -o yaml > clusters.yaml
+kubectl get kubermaticsettings -A -o yaml > kubermaticsettings.yaml
+kubectl get presets -A -o yaml > presets.yaml
+kubectl get projects -A -o yaml > projects.yaml
+kubectl get seeds -A -o yaml > seeds.yaml
+kubectl get users -A -o yaml > users.yaml
+kubectl get userprojectbindings -A -o yaml > userprojectbindings.yaml
+kubectl get usersshkeies -A -o yaml > usersshkeies.yaml
+kubectl get verticalpodautoscalers -A -o yaml > verticalpodautoscalers.yaml
+```
+
+Administrators should also check each seed cluster's Minio to verify that etcd snapshots for
+all user clusters are in place.
+
+#### Convert `datacenters.yaml` / Helm Values
+
+The Kubermatic Installer provides commands to automatically convert the old configuration files
+into their new formats.
+
+The `convert-helm-values` command takes a Helm values.yaml file and outputs (to stdout) a number
+of YAML files, representing the generated KubermaticConfiguration, Seeds, Secrets, Presets etc.
+Using this command is recommended, as it does all conversions in a single step, because the old
+`values.yaml` contained all information in some form or another. Make sure to add `--pause-seeds`
+to automatically add the `operator.kubermatic.io/skip-reconciling` annotation.
+
+```bash
+./kubermatic-installer convert-helm-values --pause-seeds myvalues.yaml
+# apiVersion: operator.kubermatic.io/v1alpha1
+# kind: KubermaticConfiguration
+# metadata:
+#   name: kubermatic
+#   namespace: kubermatic
+# ...
+#
+# ---
+# apiVersion: kubermatic.k8s.io/v1
+# kind: Seed
+# ...
+#
+# ---
+# ...
+```
+
+The annotation can also be applied and removed from Seed resources using `kubectl`:
+
+```
+# to add annotation
+kubectl annotate seed your-seed operator.kubermatic.io/skip-reconciling=
+
+# to remove it
+kubectl annotate seed your-seed operator.kubermatic.io/skip-reconciling-
+```
+
+In case migrating the Helm values.yaml is not feasable, the installer also offers a dedicated
```

Comment:
In which circumstances it should not be feasible?

Reply:
For some users it might be easier to just convert the datacenters to Seeds, e.g. EE users who will stay on the old Kubermatic chart, but already migrate the dynamic Seeds. Sure they could wrap their datacenters.yaml in a Helm values and then convert that, but offering a simple command that re-uses 90% of our conversion logic (except for the base64 decoding and YAML unmarshalling) seems like a quick and easy win.